zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/mapper/search/SearchQueryResponseMapper.java

```diff
@@ -709,6 +710,12 @@ public static BatchOperationResponse toBatchOperation(final BatchOperationEntity
         .batchOperationType(BatchOperationTypeEnum.fromValue(entity.operationType().name()))
         .startDate(formatDate(entity.startDate()))
         .endDate(formatDate(entity.endDate()))
+        .actorType(
+            ofNullable(entity.actorType())
+                .map(Enum::name)
+                .map(BatchOperationActorTypeEnum::fromValue)
+                .orElse(null))
```

Comment:
❓ For the other enums above (`batchOperationType` and `state`), we just use `.<attribute>(<REST-enum>.fromValue(entity.<enumAttribute>().name()))`. Should we do the same here? If there is an inherent benefit in doing it the way you did now, should we adjust how we handle the other enum attributes?